src/Tools/ConsoleRunner.php

```diff
@@ -103,6 +116,43 @@ private function handleHelpCommand(): string
             $this->dumpSupportedCommands($this->commandDesccriptions);
     }
 
+    /**
+     * @param array<string, string> $arguments
+     */
+    private function handleSchemaCommand(array $arguments, ClassLoader $autoloader): string
+    {
+        if (isset($arguments[self::ARGUMENT_HELP])) {
+            return $this->dumpHelpHeader() .
+                $this->dumpSupportedParameters($this->argumentDescriptions[self::COMMAND_SCHEMA]);
+        }
+
+        $allViewHelpers = (new ViewHelperFinder())->findViewHelpersInComposerProject($autoloader);
+
+        $groupedByNamespace = [];
+        foreach ($allViewHelpers as $viewHelper) {
+            $groupedByNamespace[$viewHelper->xmlNamespace] ??= [];
+            $groupedByNamespace[$viewHelper->xmlNamespace][] = $viewHelper;
+        }
+
+        $destination = (isset($arguments[self::ARGUMENT_DESTINATION])) ? rtrim($arguments[self::ARGUMENT_DESTINATION], '/') . '/' : './';
```

Comment:
I'd suggest to rather add the slash on line 151. It makes this line simpler:

`$destination = $arguments['destination'] ?? '.';`